indico/modules/events/management/controllers/protection.py

```diff
@@ -120,7 +119,7 @@ def _update_session_coordinator_privs(self, form):
         update_session_coordinator_privs(self.event, data)
 
     def _update_subcontrib_settings(self, form):
-        subcontribution_settings.set(self.event, 'speakers_can_submit', form.subcontrib_speakers.data)
+        self.event.speakers_can_submit = form.subcontrib_speakers.data
```

Comment:
Remove the method here and handles this in `update_event_protection`. You also need to handle logging of the change in there by adding it to the `log_fields` in that util.